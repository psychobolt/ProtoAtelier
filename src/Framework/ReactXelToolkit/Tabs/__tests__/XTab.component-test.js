import React from 'react';
import { shallow } from 'enzyme';

import XTab from '../XTab.component';

describe('components <XTab />', () => {
  it('should render correctly -- active tab', () => {
    const props = {
      active: true,
      children: <span>Link</span>,
    };
    const wrapper = shallow(<XTab {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly -- inactive tab', () => {
    const props = {
      children: <span>Link</span>,
    };
    const wrapper = shallow(<XTab {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
