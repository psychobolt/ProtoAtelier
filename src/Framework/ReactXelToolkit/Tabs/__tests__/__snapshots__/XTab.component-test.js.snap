// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <XTab /> should render correctly -- active tab 1`] = `
<x-tab
  selected={true}
>
  <x-label>
    <span>
      Link
    </span>
  </x-label>
</x-tab>
`;

exports[`components <XTab /> should render correctly -- inactive tab 1`] = `
<x-tab
  selected={null}
>
  <x-label>
    <span>
      Link
    </span>
  </x-label>
</x-tab>
`;
